lib/number.js

```diff
@@ -137,16 +137,25 @@ class Number {
               var m1 = line.match(/^\$\$\$(.*?isec)\$\$\$$/);
               if (m1) this.tableofcontents(this.toc[m1[1]]?.sub || [], out, "");
               else {
+                m1 = line.match(/^\$\$\$include\s+(.*?)\$\$\$$/);
```

Comment:
No trailing whitespace allowed after the closing `$$$`? That could be hard to spot.

Reply:
If the images are embedded in the Markdown and HTML files, then we don't need to copy the `images` subfolder to `/docs`.

Reply:
Perhaps we also need *real* images later.